python/smqtk/web/search_app/modules/iqr/templates/base.html

```diff
@@ -1 +0,0 @@
-../../../templates/base.html
```

Comment:
This previous symlink seems to have been replaced with a hard file in the current branch. E.g. `smqtk_iqr/web/search_app/modules/iqr/templates/base.html` should reference `smqtk_iqr/web/search_app/templates/base.html` as they are the same file.